src/cljs/harja/tiedot/urakka/suunnittelu/mhu_kustannussuunnitelma.cljs

```diff
@@ -2529,11 +2529,14 @@
         (assoc-in [:kattohinta :grid 0] (merge {:rivi :kattohinta}
                                           (into {} (map (fn [{:keys [kattohinta hoitokausi]}]
                                                           {(keyword (str "kattohinta-vuosi-" hoitokausi))
-                                                           kattohinta}) vastaus))))
+                                                           (or kattohinta 0)}) vastaus))))
```

Comment:
oisko näille indeksinumeroille näppärä määrittää jotkut symbolit, jotka kuvaa, mitä siinä indeksissä on? vois olla selkeämpi, jos ne on näin hardkoodattu

Reply:
Hyvä pointti. Testailin vähän, ja näyttäisisi siltä, että tuossa voisi käyttää indeksejen sijasta ihan vaan avaimia, etenkin nyt kun niitä ei lisäillä tai poistella.